shoot.sh

```diff
@@ -1 +1 @@
-curl 127.0.0.1:16726/order/1337/price/5
+curl 127.0.0.1:16726/order/1111/price/30
```

Comment:
thats not the way it reads, something like that instead:
`curl -X POST 'http://127.0.0.1:16726/v2/order/1111?currency=USDC&amount=30&callback=https://example.com/callback'`